fendermint/actors/bucket/src/state.rs

```diff
@@ -701,4 +705,61 @@ mod tests {
         let result = result.unwrap();
         assert_eq!(result.0.len(), 1);
     }
+
+    #[test]
+    fn test_list_with_prefix_and_without_and_limit() {
```

Comment:
This test somehow illustrates the problem. In `main`, it fails, because one of the `list` results in `0` elements